examples/github/deploy-to-a-single-device.yml

```
@@ -0,0 +1,60 @@
+#
+# The example shows how to deploy a Mender artifact to a single device.
+#
+# 'MENDER_SERVER_ACCESS_TOKEN' secret is required to be set in GitHub repo settings
+#
+
+name: Build and Deploy to a Single Device
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  MENDER_SERVER_URL: https://hosted.mender.io           # Mender server URL
+  MENDER_DEPLOYMENT_NAME: my-app-v1.0-dev               # Mender deployment name
+  MENDER_RELEASE_NAME: change-the-world-to-better-v1.0  # Mender release name
+  MENDER_ARTIFACT_NAME: artifact.mender                 # Mender artifact name
+  MENDER_DEVICES_LIST: '[\"5ca6090ff3a8e90001206535\"]' # List of devices to deploy built artifact to
```

If this is a regular shell variable, we don't need to escape the quotes
```suggestion
  MENDER_DEVICES_LIST: '["5ca6090ff3a8e90001206535"]' # List of devices to deploy built artifact to
```

---

Okey